src/DataSet.cpp

```diff
@@ -245,7 +246,16 @@ int DataSet::readPsm(const std::string& line, const unsigned int lineNr,
     }
     proteins.swap(myPsm->proteinIds); // shrink to fit
   }
-  
+
+  if (label == -1) {
+    for(auto const& value: myPsm->proteinIds) { 
+    std::string token = value.substr(0, value.find("_") + 1);
+    
+    if (token != decoyPrefix && VERB > 1) {
+      std::cerr << "Warning: Set decoy prefix don't match" << std::endl;
+    }
+  }
+  }
```

Comment:
Check indentation, also further down in the file

Reply:
Fixed